### PR TITLE
chore: add sveltekit example

### PR DESCRIPTION
**Pull request by enesozturk**
# Description

Adds SvelteKit example app with Ethers adapter

## Type of change

- [x] Chore (non-breaking change that addresses non-functional tasks, maintenance, or code quality improvements)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Associated Issues

For Linear issues: Closes APKT-xxx
For GH issues: closes #...

# Showcase (Optional)

If there is a UI change include the screenshots with before and after state.
If new feature is being introduced, include the link to demo recording.

# Checklist

- [ ] Code in this PR is covered by automated tests (Unit tests, E2E tests)
- [ ] My changes generate no new warnings
- [ ] I have reviewed my own code
- [ ] I have filled out all required sections
- [ ] I have tested my changes on the preview link
- [ ] Approver of this PR confirms that the changes are tested on the preview link

### CODE REVIEW

**Review by magiziz — APPROVED** [review]

File: examples/sveltekit-ethers/svelte.config.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+const config = {
+  // Consult https://svelte.dev/docs/kit/integrations
+  // for more information about preprocessors
+  preprocess: vitePreprocess(),
+
+  kit: {
+    // adapter-auto only supports some environments, see https://svelte.dev/docs/kit/adapter-auto for a list.
+    // If your environment is not supported, or you settled on a specific environment, switch out the adapter.
+    // See https://svelte.dev/docs/kit/adapters for more information about adapters.
+    adapter: adapter()
```
Comment (tomiir) [inline examples/sveltekit-ethers/svelte.config.js:14]: why is this required here?  have we tried removing?

Comment (enesozturk) [reply examples/sveltekit-ethers/svelte.config.js:14]: Comes with SvelteKit template - will remove since we don't use adapters feature
